packages/11ty/_plugins/iiif/process/copyManifest.js

```diff
@@ -0,0 +1,46 @@
+const fs = require('fs')
+const path = require('path')
+const getId = require('./getId')
+
+/**
+ * @param  {Object} config Quire IIIF Process config
+ * @return {Function}      copyManifest()
+ */
+module.exports = (config) => {
+  const { 
+    output: defaultOutput,
+    manifestFilename,
+    root
+  } = config
+
+/**
+ * Copies manifest from iiif input directory to output directory
+ * Overwrites existing manifest if lazy === false
```

Comment:
@hbalenda can this module be replaced by using the [Eleventy Passthrough File Copy](https://www.11ty.dev/docs/copy/) feature?